upgrade/helpers/tasks.py

```diff
@@ -727,3 +728,46 @@ def puppet_autosign_hosts(version, hosts, append=True):
         'ver2': '/etc/puppetlabs/puppet/autosign.conf'}
     for host in hosts:
         run('echo "{0}" {1} {2}'.format(host, append, puppetfile[puppetver]))
+
+
+def get_satellite_host():
```

Comment:
We already have this, somewhere . why duplicate

Reply:
https://github.com/SatelliteQE/satellite6-upgrade/blob/63ddfad7563ffba4c862458a3b5dc11145f65545/upgrade_tests/helpers/scenarios.py#L132

Reply:
I want to remove it from there and keep it here.

Reply:
I changes that everywhere